src/objects/media/media.stories.mdx

```diff
@@ -46,49 +47,9 @@ const imageDemoTransformSource = (_src, storyContext) => {
     reverse: false,
   }}
   argTypes={{
+    ...mediaArgTypes,
     imgSrc: { type: { name: 'string', required: true } },
```

Comment:
I didn't move this `argType` because it is related to the demo, not the template itself.